tests/rules/target/naming-convention/invalid/class_property_not_as_filter.ts

```diff
@@ -0,0 +1,8 @@
+class PropertyCasingInvalidByFilter {
+  _id: string
+  _id_: string
+  _id_id: string
+  __v: string
+  ___v: string
+  __v_v: string
+}
```

Comment:
이 폴더를 본 사람들이 이렇게 쓰면 맞게 쓰는거구나 생각할 수 있게 하고자 하는 부분도 있었는데요. 
그런 의미에서 정상적인 케이스는 제외해주는게 어떨까요?

```suggestion
class PropertyCasingInvalidByFilter {
  _id_: string
  _id_id: string
  ___v: string
  __v_v: string
}
```